docs/source/_static/custom.css

```diff
@@ -0,0 +1,9 @@
+/* 24px margin from readthedocs theme */
+div.nblast {
+    margin-bottom: 19px !important;  /* padding has already 5px */
```

Comment:
I don't know, I couldn't find one.

But I think in this case it doesn't matter, because there will be no further CSS "derived" from this.

Anyway, if somebody knows a way, I'm happy to change this!